lib/defaults.js

```diff
@@ -2,260 +2,111 @@
 
 module.exports = {
   extends: 'airbnb-base/legacy',
-  'env': {                           // http://eslint.org/docs/user-guide/configuring.html#specifying-environments
-    'browser': true,                 // browser global variables
-    'node': true,                    // Node.js global variables and Node.js-specific rules
-    'es6': true
+
+  env: {                           
+    browser: true, // Browser global variables
+    node: true, // Node.js global variables and Node.js-specific rules
+    es6: true
   },
-  'plugins': [
+
+  plugins: [
     'virtru-lint'
   ],
 
-/*@TODO: Decide upon these later
```

Comment:
This file was extremely messy, so I removed unused sections like this. We have a separate ES6 (https://github.com/virtru/lint-trap/tree/master/lib/es6) section in this repo where this section can go in the future if relevant.